priv/repo/seeds.exs

```diff
@@ -9,3 +9,12 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+require Logger
```

Comment:
we shall the prod livebook for handling seeding functions on the prod env 



this seeds.exs currently helps with setting up a new repo, it's the minio setup that it's doing